config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/builder/IdentityProviderEntityBuilder.java

```diff
@@ -35,25 +38,27 @@
     }
 
     private Entity buildIdentityProviderEntity(String name, IdentityProvider identityProvider) {
-        final Element identityProviderElement = document.createElement("l7:IdentityProvider");
+        final Element identityProviderElement = document.createElement(ID_PROV);
```

Comment:
wait for my changes to get merged first, then you can use createElementWithAttributesAndChildren :)